### PR TITLE
ortp 5.2.51

### PR DESCRIPTION
**Pull request by ZhongRuoyu**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

- [x] `resource` blocks have been checked for updates.

### CODE REVIEW

File: Formula/ortp.rb [hunk 1]
```diff
@@ -24,8 +24,8 @@ class Ortp < Formula
   # https://github.com/BelledonneCommunications/bctoolbox
   resource "bctoolbox" do
     # Don't forget to change both instances of the version in the URL.
-    url "https://gitlab.linphone.org/BC/public/bctoolbox/-/archive/5.2.49/bctoolbox-5.2.49.tar.bz2"
-    sha256 "4f4f76624115d0420ee0dfeb4bff51ca9d1339822ebc921c11f040cde3e86a9d"
+    url "https://gitlab.linphone.org/BC/public/bctoolbox/-/archive/5.2.49/bctoolbox-5.2.51.tar.bz2"
```
Comment (carlocab) [inline Formula/ortp.rb:27]:
```suggestion
    url "https://gitlab.linphone.org/BC/public/bctoolbox/-/archive/5.2.51/bctoolbox-5.2.51.tar.bz2"
```

Comment (ZhongRuoyu) [reply Formula/ortp.rb:27]: Sorry 😅

Comment (carlocab) [reply Formula/ortp.rb]: > ```
> # Don't forget to change both instances of the version in the URL.
> ```
😠

**Review by carlocab — APPROVED** [review]

File: Formula/ortp.rb [hunk 2]
```diff
@@ -24,8 +24,8 @@ class Ortp < Formula
   # https://github.com/BelledonneCommunications/bctoolbox
   resource "bctoolbox" do
     # Don't forget to change both instances of the version in the URL.
-    url "https://gitlab.linphone.org/BC/public/bctoolbox/-/archive/5.2.49/bctoolbox-5.2.49.tar.bz2"
-    sha256 "4f4f76624115d0420ee0dfeb4bff51ca9d1339822ebc921c11f040cde3e86a9d"
+    url "https://gitlab.linphone.org/BC/public/bctoolbox/-/archive/5.2.51/bctoolbox-5.2.51.tar.bz2"
```
Comment (carlocab) [inline Formula/ortp.rb:27]:
```suggestion
    url "https://gitlab.linphone.org/BC/public/bctoolbox/-/archive/#{version}/bctoolbox-#{version}.tar.bz2"
```
I think this helps make sure no one else makes the same mistake in the future! We'll get checksum mismatches from `bump-formula-pr`, but that's not really undesirable.

Comment (ZhongRuoyu) [reply Formula/ortp.rb:27]: Rare in Homebrew/core, but sounds good.

**Review by carlocab — APPROVED** [review]

File: Formula/ortp.rb (hunk 2 above)
Comment (ZhongRuoyu) [reply Formula/ortp.rb]: > no one else

I'm pretty sure this will prevent myself from doing so, too. 😬 I'll be more careful.

Comment (ZhongRuoyu) [reply Formula/ortp.rb]: Ah, that didn't work. The `resource` has its own version.

**Pull request merged**